src/UnderlineNav2/UnderlineNavItem.tsx

```diff
@@ -70,30 +77,51 @@ export const UnderlineNavLink = forwardRef(
     const textStyles: BetterSystemStyleObject = {
       whiteSpace: 'nowrap'
     }
+
+    const wrapperStyles = {
+      display: 'inline-flex',
+      paddingY: 1,
+      paddingX: 2
+    }
     const smallVariantLinkStyles = {
-      paddingY: 2,
+      paddingY: 1,
       fontSize: 0
     }
     const defaultVariantLinkStyles = {
-      paddingY: 3,
+      paddingY: 2,
       fontSize: 1
     }
 
     const linkStyles = {
       display: 'inline-flex',
       color: 'fg.default',
       textAlign: 'center',
-      borderBottom: '2px solid transparent',
-      borderColor: selectedLink === ref ? 'primer.border.active' : 'transparent',
       textDecoration: 'none',
-      paddingX: 2,
+      paddingX: 1,
       marginRight: 3,
       ...(variant === 'small' ? smallVariantLinkStyles : defaultVariantLinkStyles),
-      '&:hover, &:focus': {
-        borderColor: selectedLink === ref ? 'primer.border.active' : 'neutral.muted',
+      '&:hover > div[data-component="wrapper"] ': {
+        backgroundColor: 'neutral.muted',
+        borderRadius: 2
+      },
+      '&:focus': {
+        outlineColor: 'accent.fg',
+        borderRadius: 2,
+        outlineOffset: '-6px',
```

Comment:
I don't think we can set X and Y outline offsets ([ref](https://developer.mozilla.org/en-US/docs/Web/CSS/outline-offset)) , so this is the limitation came from using native outline.  (Not super close to the design -  see the x paddings are less in the design)

### Design
<img width="197" alt="Screen Shot 2022-08-29 at 3 33 57 pm" src="https://user-images.githubusercontent.com/1446503/187129844-9a9c9bf5-7884-4d80-beff-b866c0bc9b0c.png">

### Storybook
<img width="190" alt="Screen Shot 2022-08-29 at 3 40 39 pm" src="https://user-images.githubusercontent.com/1446503/187130582-b0343149-ccce-4b2b-b357-6d1693af0fa0.png">

Reply:
with the new commit this is not relevant anymore